loops.js

```diff
@@ -50,4 +59,9 @@
 // >> 8
 // >> 9
 // ==========================================
+for (let i = 0; i <=9; i++){
+    if (i<=3){console.log(i)}
+    else if(i<=6){console.log(">"+i) }
+    else{console.log(">>"+i)}}
+
```

Comment:
Hey je code logt niet zoals in het voorbeeld.

Dit had ik;

for ( let i = 0; i < 10; i++ )
    if  (i > 5 ) { console.log ( ">> " + i ); }
    else if  ( i > 2 ) { console.log ("> "+ i ); }
    else { console.log  (i + " ") }